Common/WALinuxAgent-2.0.16/waagent

```diff
@@ -741,6 +741,29 @@ class AbstractDistro(object):
         pass
 
 
+class DefaultDistro(AbstractDistro):
+    """
+    Default Distro concrete class: This class serves as a default OS behavior class.
+    """
+
+    def startDHCP(self):
```

Comment:
How much functionality do we lose by having these functions be noop?

Reply:
I validated - `startDHCP` and `stopDHCP` are not used by any extension directly, and the implementation is only used in the `Agent.Run`, but we don't ever call that (that is the way to run the old agent).